### PR TITLE
Ignore load error exception to prevent stopping internal heartbeat fo…

### PR DESCRIPTION
**Pull request by wtao-mux**
…r v2.7.3 and v2.8.0 exoplayer

### CODE REVIEW

**Comment by wtao-mux** [comment]
only for v2.7.3 and v2.8.0 exoplayer. The previous versions won't trigger onLoadError event.

File: MuxExoPlayer/src/r2_7_3/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java [hunk 1]
```diff
@@ -255,7 +255,6 @@ public void onLoadCanceled(DataSpec dataSpec, int dataType, int trackType, Forma
 
     @Override
     public void onLoadError(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason, Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs, long loadDurationMs, long bytesLoaded, IOException e, boolean wasCanceled) {
-        internalError(new MuxErrorException(ERROR_IO, "IOException - " + e.getMessage()));
```
Comment (jsanford8) [inline MuxExoPlayer/src/r2_7_3/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java:258]: Should we not do this for all Exo versions, just to be consistent across them all?

Comment (wtao-mux) [reply MuxExoPlayer/src/r2_7_3/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java:258]: i tested v2.6.0. it won't call onLoadError. but,...yeah, you are right, there is no harm to change all. i will make another commit

**Comment by wtao-mux** [comment]
interesting. in r2.04, r2.1.1, r2.2.0, r2.3.1, r2.5.1. and r2.6.0 exoplayer listeners, there is no internalException invocation. 

so in summary, r2.4.1, r2.7.3, and r2.8.0 are affected and fixed.

**Review by jsanford8 — APPROVED** [review]
lgtm.

Again weird about the build failing, though...

**Pull request merged**